tests/FSharp.Compiler.Service.Tests/MultiProjectAnalysisTests.fs

```diff
@@ -557,24 +562,23 @@ let ``Test multi project symbols should pick up changes in dependent projects``
     let wt0b = System.DateTime.UtcNow
     let wt1b = FileSystem.GetLastWriteTimeShim MultiProjectDirty1.fileName1
     printfn "Writing old content to file '%s'" MultiProjectDirty1.fileName1
-    System.Threading.Thread.Sleep(1000)
+    //System.Threading.Thread.Sleep(1000)
     FileSystem.OpenFileForWriteShim(MultiProjectDirty1.fileName1).Write(MultiProjectDirty1.content)
     printfn "Wrote old content to file '%s'"  MultiProjectDirty1.fileName1
     let wt2b = FileSystem.GetLastWriteTimeShim MultiProjectDirty1.fileName1
     printfn "Current time: '%A', ticks = %d"  wt0b wt0b.Ticks
     printfn "Old write time: '%A', ticks = %d"  wt1b wt1b.Ticks
     printfn "New write time: '%A', ticks = %d"  wt2b wt2b.Ticks
 
-    count.Value |> shouldEqual 4
+    count |> shouldEqual 4
     let wholeProjectResults2AfterChange2 = checker.ParseAndCheckProject(proj2options) |> Async.RunImmediate
 
-    System.Threading.Thread.Sleep(1000)
-    count.Value |> shouldEqual 6 // note, causes two files to be type checked, one from each project
+    count |> shouldEqual 6 // note, causes two files to be type checked, one from each project
```

Comment:
With Transparent Compiler the check count sometimes stops at 5.

Reply:
This should be now sorted out now. Apparently it had something to do with reusing the same physical files among test cases.